raiden/tests/test_files/synapse-config.yaml

```diff
@@ -39,7 +39,7 @@ max_upload_size: "0M"  # changed
 max_image_pixels: "0M"  # changed
 url_preview_enabled: false
 
-enable_registration: true  # changed
```

Comment:
Same as above (even though this file already exists) I think it would be better located in `tools/`.

Reply:
@ulope Done